backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepository.java

```diff
@@ -320,6 +324,11 @@ public Optional<OrganizationRoleClaims> getOrganizationRoleClaimsForUser(String
       }
       return Optional.ofNullable(usernameOrgRolesMap.get(username));
     } catch (ScopeNotActiveException e) {
+      // Tests are set up with a full SecurityContextHolder and should not rely on
+      // usernameOrgRolesMap as the source of truth.
+      if (!environment.equals("UNITTEST") && usernameOrgRolesMap.containsKey(username)) {
```

Comment:
This is not my preferred way to go about this, but the default entities set up for testing don't map well to this orgRolesMap. Without the extra check, unit tests fail because they're not configured to the right roles.